core/Dockerfile.template

```diff
@@ -24,7 +24,11 @@ COPY --from=npm-install /tmp/node ./
 
 RUN install_packages python3 make build-essential openssh-client
 
-RUN npm install balena-cli@12.51.1 -g --production --unsafe-perm
+RUN curl -L https://github.com/balena-io/balena-cli/releases/download/v12.51.1/balena-cli-v12.51.1-linux-x64-standalone.zip -o balena-cli.zip \
```

Comment:
@vipulgupta2048 this binary is for x64 and won't run on the Fin.

Reply:
Ahh so that's why we have an NPM install in place here, and so do you in your balena-cli base image. I gotcha.

Reply:
Will wait for the balena-cli to releae the apt installation method.